Sources/ClientRuntime/Networking/Http/Middlewares/SignerMiddleware.swift

```diff
@@ -57,6 +57,11 @@ extension SignerMiddleware: ApplySigner {
             updatedSigningProperties.set(key: AttributeKey(name: "SignedBodyValue"), value: bodyValue)
         }
 
+        if case .stream(let stream) = request.body, stream.isEligibleForChunkedStreaming {
+            // Pass in context object via signing properties to reuse final checksum value in chunked streaming.
+            updatedSigningProperties.set(key: AttributeKey(name: "Context"), value: attributes)
```

Comment:
Be careful here... if we put Context into signing properties then we put those signing properties into Context, a reference cycle is formed and a memory leak will occur.

Maybe better to copy the specific things we need from Context into signingProperties?